src/core_plugins/kibana/server/tutorials/elasticsearch_metrics/index.js

```diff
@@ -17,23 +17,36 @@
  * under the License.
  */
 
+import { i18n }  from '@kbn/i18n';
 import { TUTORIAL_CATEGORY } from '../../../common/tutorials/tutorial_category';
 import { onPremInstructions, cloudInstructions, onPremCloudInstructions } from '../../../common/tutorials/metricbeat_instructions';
 
 export function elasticsearchMetricsSpecProvider() {
   const moduleName = 'elasticsearch';
   return {
     id: 'elasticsearchMetrics',
-    name: 'Elasticsearch metrics',
+    name: i18n.translate('kbn.server.tutorials.elasticsearchMetrics.nameTitle', {
+      defaultMessage: 'Elasticsearch metrics',
+    }),
     isBeta: true,
     category: TUTORIAL_CATEGORY.METRICS,
-    shortDescription: 'Fetch internal metrics from Elasticsearch.',
-    longDescription: 'The `elasticsearch` Metricbeat module fetches internal metrics from Elasticsearch.' +
-                     ' [Learn more]({config.docs.beats.metricbeat}/metricbeat-module-elasticsearch.html).',
+    shortDescription: i18n.translate('kbn.server.tutorials.elasticsearchMetrics.shortDescription', {
+      defaultMessage: 'Fetch internal metrics from Elasticsearch.',
+    }),
+    longDescription: i18n.translate('kbn.server.tutorials.elasticsearchMetrics.longDescription', {
+      // eslint-disable-next-line no-multi-str
+      defaultMessage: 'The `elasticsearch` Metricbeat module fetches internal metrics from Elasticsearch. \
+[Learn more]({learnMoreLink}).',
+      values: {
+        learnMoreLink: '{config.docs.beats.metricbeat}/metricbeat-module-elasticsearch.html',
```

Comment:
Will this link always lead to the EN version of the Elastic site?
https://www.elastic.co/guide/en/beats/metricbeat/6.3/metricbeat-module-elasticsearch.html?

Answer is not stopping this PR. I'm just curious if the i18n framework can change a URL part based on locale

Reply:
@azasypkin maybe you could help with clarification?

Reply:
There is a high chance that the set of locales we support in Kibana and have documentation for will be different so I'd not rely on anything "automatic" here, but we can add `context` field where we ask localizers to ping us and check whether we have docs for the locale they are currently working on so that every time we generate these labels for vendor to translate they will see that context field and can get back to us and clarify.